integration-tests/__tests__/lifecycles.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+'use strict';
+
+const runJest = require('../runJest');
+
+test('suite with invalid assertions in afterAll', () => {
+  const {stderr, status} = runJest('lifecycles');
+
+  expect(status).toBe(1);
+  expect(stderr).toMatch(/FAIL/);
```

Comment:
can you assert that the original error is in here as well?

Reply:
Sure :)